qualtricssurvey/private/view.js

```diff
@@ -0,0 +1,11 @@
+function QualtricsSurveyView(runtime, element) {
```

Comment:
I could just remove `jsView` from `concat` (and remove this file, of course). I believe everything else just searches for `.js` files and only does anything if `.js` files are found.